pom.xml

```diff
@@ -3,12 +3,12 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.bina.lrsim</groupId>
-    <artifactId>lrsim</artifactId>
-    <version>0.1</version>
+    <artifactId>LongISLND</artifactId>
+    <version>0.9</version>
```

Comment:
@johnmu @marghoob this should change `target/lrsim-0.1.jar` to `target/LongISLND-0.9.jar`, would there be any side effects outside of the `target` directory?